server/pbenchinacan/load_keycloak.sh

```diff
@@ -26,13 +26,17 @@ ADMIN_PASSWORD=${ADMIN_PASSWORD:-"admin"}
 REALM=${KEYCLOAK_REALM:-"pbench-server"}
 CLIENT=${KEYCLOAK_CLIENT:-"pbench-client"}
 
+PB_DEPLOY_FILES=${PB_DEPLOY_FILES:-"/var/tmp/pbench/pbench_server_deployment"}
```

Comment:
This default value isn't quite consistent with the one in the higher-level script, and I think it should be.